### PR TITLE
[Fix] Add client redirects

### LINKED ISSUE #135: Add client redirects in keycloak

### PR DESCRIPTION
**Pull request by MattDodsonEnglish**
Fix #135

### CODE REVIEW

File: content/deploy/install/keycloak.md [hunk 1]
```diff
@@ -63,6 +63,12 @@ In Keycloak, _clients_ are entities that request Keycloak to authenticate a user
 You need to create a client for each service.
 
 The DB client requires additional configuration of flows and grants.
+Other clients, such as the UI and Dashboard, use the standard flow to coordinate authorization between the browser and Keycloak to simplify security and improve user convenience.
+
+{{< notice "note" >}}
+Each client has its own subdomain.
```
Comment (tomhollingworth) [inline content/deploy/install/keycloak.md:69]: Each standard flow client...


Our DB, Core, BPMN, Audit, Agent clients don't have subdomains.

**Review by tomhollingworth — CHANGES_REQUESTED** [review]

File: content/deploy/install/keycloak.md [hunk 2]
```diff
@@ -116,13 +122,42 @@ Create a client for the UI as follows:
...
+   - For **Authentication flow**, enable:
+     - 🗸 Standard flow
+     - 🗸 Direct access grants
+     - 🗸 Implicit flow
+
+1. Configure the **Access Settings**:
+
+   - **Root URL**: `<DASHBOARD_SUBDOMAIN>.<YOUR_DOMAIN>` without trailing slashes
+   - **Home URL**: `<DASHBOARD_SUBDOMAIN>.<YOUR_DOMAIN>` without trailing slashes
+   - **Valid redirect URIs**: `<DASHBOARD_URL>/login/generic_oath` without trailing slashes
```
Comment (tomhollingworth) [inline content/deploy/install/keycloak.md:156]: typo in `/generic_oauth`

Is there any difference between `<DASHBOARD_URL>` and `<DASHBOARD_SUBDOMAIN>.<YOUR_DOMAIN>` ?

I'd expect it to mirror  the others with this trailing path i.e. `<DASHBOARD_SUBDOMAIN>.<YOUR_DOMAIN>/login/generic_oauth`

Comment (MattDodsonEnglish) [inline content/deploy/install/keycloak.md:156]:
```suggestion
   - **Valid redirect URIs**: `<DASHBOARD_URL>/login/generic_oauth` without trailing slashes
```

Comment (MattDodsonEnglish) [inline content/deploy/install/keycloak.md:156]:
```suggestion
   - **Valid redirect URIs**: `<DASHBOARD_SUBDOMAIN>.<YOUR_DOMAIN>/login/generic_oath` without trailing slashes
```

Comment (MattDodsonEnglish) [reply content/deploy/install/keycloak.md]: Good catches here, thanks!

**Pull request merged**